incubation.md

```diff
@@ -0,0 +1,130 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which certain types of new Jupyter Subprojects are created and
```

Comment:
I would probably phrase this as

> Incubation is _a_ process by which new Jupyter Subprojects _can be_ created...

avoiding "certain types" which lacks a definition, and avoiding the impression that this isn't the one and only (or even primary) way projects are added.